modules/claims_api/config/schemas/v2/request_bodies/disability_compensation/request.json

```diff
@@ -3,1080 +3,1097 @@
   "description": "Claims API 526 Schema",
   "type": "object",
   "additionalProperties": false,
-  "required": [
-    "veteranIdentification",
-    "serviceInformation",
-    "disabilities",
-    "claimantCertification",
-    "claimProcessType"
-  ],
+  "required": ["data"],
   "properties": {
-      "claimDate": {
-          "pattern": "^(\\d{4})-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])$",
-          "type": "string",
-          "example": "2018-06-04"
-      },
-      "claimProcessType": {
-          "type": "string",
-          "description": "Type of claim process.",
-          "enum": [
-              "STANDARD_CLAIM_PROCESS",
-              "FDC_PROGRAM",
-              "BDD_PROGRAM_CLAIM",
-              "TBD"
-          ]
-      },
-      "claimantCertification": {
-          "type": "boolean",
-          "description": "Claimant certifies and authorizes release of information.",
-          "default": false
-      },
-      "veteranIdentification": {
-          "description": "Veteran Object being submitted in Claim",
-          "type": "object",
-          "additionalProperties": false,
-          "required": ["mailingAddress", "currentlyVAEmployee"],
+    "data": {
+      "type":"object",
+      "required":["attributes"],
+      "properties": {
+        "attributes": {
```

Comment:
I think we want to add `type` here as another property, but v1 doesn't have it so I'm going to approve this and we can circle back if we want it to align. It may actually cause an issue with the OAS too? curious - maybe that's why it's not in v1. 🤔